java/demo/src/com/google/phonenumbers/PhoneNumberParserServlet.java

```diff
@@ -188,6 +243,9 @@ private String getNewIssueLink(String phoneNumber, String defaultCountry)
         "Supporting evidence (for example, national numbering plan, announcement from mobile "
         + "carrier, news article): **IMPORTANT - anything posted here is made public. "
         + "Read the guidelines first!** \n\n");
+    issueTemplate.append("[link to demo]("
```

Comment:
This is linking from a github issue back to the demo result page for the number which the issue is about. That should be really useful for us.